en/asgardeo/docs/tutorials/smart-on-fhir.md

```diff
@@ -0,0 +1,256 @@
+
+We have created a Postman collection to automate the following process:
+
+- Get a bearer token to access Asgardeo REST APIs
+
+- Create the `fhirUser` user attribute. This will be used as the identity of a user accessing FHIR resources.
+
+- Create the `fhirUser` scope. Applications can request this scope to access the `fhirUser` attribute of a user.
+
+- Add the relevant OIDC and SCIM dialects of the `fhirUser` user attribute.
+
+- Update the `fhirUser` attribute of a selected user to `Patient/1`.
+
+- Register your SMART on FHIR app using Dynamic Client Registration (DCR).
+
+- Configure your SMART on FHIR application to request the `fhirUser` local attribute from users during login.
+
+Follow the steps below to download and run the Postman collection:
+
+1. Download the Postman collection from [Github](https://github.com/wso2-enterprise/open-healthcare/blob/main/scripts/postman-collections/smartonfhir-asgardeo.postman_collection.json){target="_blank"} and import it to Postman.
+
+2. In the Postman collection named **Asgardeo**, go to the **Authorization** tab and do the following:
+
+    1. Under **Auth Type**, select **Basic Auth**.
+
+    2. Enter the client ID and client secret of your application as username and password respectively.
+
+    ![Configure Postman authentication]({{base_path}}/assets/img/tutorials/smart-on-fhir/fhir-configure-postman-auth.png){: width="600" style="display: block; margin: 0; border: 0.3px solid lightgrey;"}
+
+3. Go to the **Variables** tab and provide the following values:
+
+    <table>
+        <tr>
+        <td>applicationName</td>
+        <td>Provide any name and this will be the name of the DCR application that will be created through the Postman script.</td>
+        </tr>
+        <tr>
+        <td>organization</td>
+        <td>Provide your organization name. You may find it in your Console URL. For example, if the URL is `https://console.asgardeo.io/t/healthcare36/app/getting-started`, the organization name is `healthcare36`.
+        </td>
+        </tr>
+        <tr>
+        <td>username</td>
+        <td>Provide the email address of the user whose `fhirUSer` attribute you want to update.</td>
+        </tr>
+    </table>
+
+4. Once the Postman collection is configured, you are ready to run it. To do so,
+
+    1. Navigate to the **Asgardeo** > **Asgardeo-configs** folder.
+
+    2. Click the three dots corresponding to **Asgardeo-configs** and click **Run folder**.
+
+    3. Ensure no errors occur and troubleshoot any errors that arise.
+
+        ![Postman test cases]({{base_path}}/assets/img/tutorials/smart-on-fhir/fhir-postman-test-cases.png)
+
+### Step 4: Verify that everything works
+
+Now that you have created the `fhirUser` user attribute and registered your SMART on FHIR app in Asgardeo, let's try to log into your application as the user and obtain the ID token. If everything has worked, the ID token should contain the `fhirUser` attribute.
+
+To do so,
```

Comment:
Shall we add the screenshot of retrieving the id-token? (The 'get-token' request's Authorization tab)